db/migrate/20240807153618_add_currency_field_to_trade.rb

```diff
@@ -0,0 +1,5 @@
+class AddCurrencyFieldToTrade < ActiveRecord::Migration[7.2]
+  def change
+    add_column :account_trades, :currency, :string
```

Comment:
Introduces a bit of denormalization here, but the alternative was quite a bit of added complexity and DB joins to read the `currency` value from `Account::Entry` for the sake of "monetizing" the `price` attribute on `Account::Trade`